docs/source/roles.md

```diff
@@ -35,8 +37,8 @@ These roles cover managing TRE computing infrastructure, but also administering
 
 * - Role name
   - Role description
-* -
-  -
+* - Output checker
+  - Person or people responsible disclosure control via checking of project outputs destined for egress from the TRE.
```

Comment:
```suggestion
  - People responsible for checking the disclosure risk of project outputs, before egress, as part of the disclosure control process.
```

I'm not sure it is correct to say the output checkers are responsible for disclosure control as a whole.